docs/configuration/overview.rst

```diff
@@ -26,9 +27,8 @@ Config File Structure
      type: push
    - ...
 
-zrepl is configured using a single YAML configuration file with two main sections: ``global`` and ``jobs``.
-The ``global`` section is filled with sensible defaults and is covered later in this chapter.
-The ``jobs`` section is a list of jobs which we are going to explain now.
```

Comment:
Did you find this line not valuable?

Reply:
No. It provides no content. Instead of saying "I am about to tell you about x on the next line', I'd say it's better to just start talking about it right away.

The first line already identifies the parts. If the reader wants to know more, they'll jump to the section anyway.